src/logging/utils.spec.ts

```diff
@@ -33,9 +33,6 @@ describe('asError', () => {
 
     const result = asError(thrown);
     expect(result).toEqual(new Error('test error'));
-
-    // If stringified:
-    expect(result).not.toEqual(new Error('{"message":"test error"}'));
```

Comment:
What about the test `should return a new Error instance with the thrown value if thrown is a string`? Shouldn't that additional assertion be removed also?